docs/quick-start.md

```diff
@@ -43,6 +43,33 @@ $ export KUBELESS_NAMESPACE=<name of namespace>
 $ export KUBELESS_CONFIG=<name of config file>
 ```
 
+or the following information can be added to `functions.kubeless.io` `CustomResourceDefinition` as `annotations`. E.g. below `CustomResourceDefinition` will signify `kubeless-controller` is installed in namespace `kubless-new-namespace` and config name is `kubeless-config-new-name`
```

Comment:
I think we are entering in too much detail here for the quick start guide. Let's move this section (From line 39 and your new text) to the file `function-controller-configuration`. We can have a section there: "Install kubeless in a different namespace" and link it here.